src/components/side-bar/side-bar.css

```diff
@@ -1,5 +1,5 @@
 .sidebar {
-  width: 10vh;
+  width: 8vh;
```

Comment:
maybe 9?
what do you think?

Reply:
אין בעיה -רק תראי איך זה נראה אצלי אחרי שאני משנה
זה מסתיר לי את הגנרל
![image](https://github.com/gemit-project/client/assets/152376943/af283650-6bd3-4916-ad40-a4cacdee1e0d)